server/src/main/java/org/elasticsearch/common/settings/AbstractScopedSettings.java

```diff
@@ -119,6 +120,13 @@ protected AbstractScopedSettings(Settings nodeSettings, Settings scopeSettings,
         settingUpdaters.addAll(other.settingUpdaters);
     }
 
+    public synchronized Settings getLoggerSettings() {
+        if (lastSettingsApplied == Settings.EMPTY) {
```

Comment:
why not get it like this?
```
Loggers.LOG_LEVEL_SETTING.get(this.lastSettingsApplied, settings);
```

Reply:
That would return the `Level`, not the `Settings` ?

Reply:
:man_facepalming: Of course, sorry